HomeWork10/HomeWork10/Services/RoomService.cs

```diff
@@ -0,0 +1,50 @@
+﻿using HomeWork10.Models;
+using HomeWork10.Services.Abstractions;
+using System.Runtime.CompilerServices;
+
+namespace HomeWork10.Services
+{
+    internal class RoomService : IRoomService
+    {
+        private readonly IRosetteService _serviceRosette;
+        private readonly Room _room;
+
+        public RoomService(IRosetteService rosetteService)
+        { 
+            _serviceRosette = rosetteService;
+            _room = new Room(); 
+        }
+       
+        public void TurnOnOffApliance(ElectricalAppliances electrical)                              
+        {
+            Console.WriteLine($"Connect to rosette {electrical.Id} : {electrical.GetType().Name} y/n?");
+           
+            _room.Appliances.Add(_serviceRosette.СonnecDisconectDevice(electrical, (Console.ReadKey().Key == ConsoleKey.Y)));
+        }
+
+        private double GetTotalElectricityCost()
+        {
+            return _room.Appliances.Where(p=>p.TornOn == true)
+                                   .Select(p=>p.Consumes)
+                                   .Sum();
+        }
+
+        public void ShowFullInfo()
```

Comment:
This name doesn't let me know anything what info for what we need this, I don't know

Reply:
agree...